zerver/lib/bot_lib.py

```diff
@@ -70,11 +69,9 @@ def send_reply(self, message, response):
                 sender_email=message['sender_email'],
             ))
 
-    def get_config_info(self, bot_name, section=None):
-        # type: (str, Optional[str]) -> Dict[str, Any]
-        conf_file_path = os.path.realpath(os.path.join(
-            our_dir, '..', 'bots', bot_name, bot_name + '.conf'))
-        section = section or bot_name
-        config = configparser.ConfigParser()
-        config.readfp(open(conf_file_path))  # type: ignore # likely typeshed issue
-        return dict(config.items(section))
+    def get_config_info(self, bot_name):
+        # type: (str) -> Dict[str, Any]
+        service_profile = get_service_profile(user_profile_id=self.user_profile.id, service_name=bot_name)
+
+        config_dict = ast.literal_eval(service_profile.service_config)
```

Comment:
We definitely don't want to do this; it's a significant security problem to run user-provided code in the Zulip Python context.  Why not just have the config be stored in JSON format and use `ujson.loads()`?  That's safe.